packages/ava/src/insight/algorithms/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './IQR';
-export * from './mannkendall-test';
+export * from './category-outlier';
+export * from './trend-direction';
```

Comment:
want to know the naming thinking

Reply:
> want to know the naming thinking

change to single file and use the lower camel case